style.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500;600;700&family=Poppins:wght@300;400;500&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap");
+
+:root {
+  --theme-color: #1c1a19;
+  --toolbar-color: #3c3a39;
+  --primary-1: #ff6b00;
+  --primary-4: #fff5e1;
+  --neutral: #dbd8d7;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+header {
+  background: var(--toolbar-color);
+  width: 100%;
+  height: 92px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 12px;
+}
+
+.nav-bar h3 {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  color: #fff;
+}
+
+.nav-bar img {
+  padding: 8px;
+  right: 24px;
+}
+
+section > .container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: var(--theme-color);
+  width: 100%;
+  min-height: 900px;
+  top: 92px;
+}
+
+.headline-title {
+  z-index: 2;
+  color: var(--primary-1);
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  text-transform: capitalize;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  margin-top: 120px;
+  width: 100%;
+  margin-left: 1.5rem;
+}
```

Comment:
It appears that your page is overflowing on mobile. This could be because of the styling applied in this section. Below is a screenshot to help you fix this.

<img width="1420" alt="screenshot" src="https://user-images.githubusercontent.com/71434803/125008847-463d4b80-e05b-11eb-904e-7b01d2902ec4.png">